data/checkoutPage.ts

```diff
@@ -24,13 +31,25 @@ export const extraFacilitationSection = {
   extraFacilitationSectionTitle: "Request meals and other services",
   facility1: "Meals",
   facility2: "Services",
+  requestText: "Special Request",
 };
 
 /* Assign title to the calendar component of the checkout page */
-export const calenderTitle = "Select your check-in and check-out dates";
+export const calenderData = {
+  calenderTitle: "Select your check-in and check-out dates",
+  termTitle: "Terms and Conditions",
```

Comment:
Add this terms and condition terms into a different object.
Let Calendar Data  has calendar specific details .